ts/screens/wallet/payment/PaymentOutcomeCodeMessage.tsx

```diff
@@ -54,12 +58,21 @@ const successFooter = (onClose: () => void) => (
 const PaymentOutcomeCodeMessage: React.FC<Props> = (props: Props) => {
   const outcomeCode = props.outcomeCode.outcomeCode.fold(undefined, oC => oC);
 
+  const { psps, verifica } = props.payment;
```

Comment:
instead of accessing the attributes from the component what about creating appropriate selectors for both of them?

Reply:
Fixed as suggested ✅